Jellyfin/Jellyfin.csproj

```diff
@@ -236,6 +236,9 @@
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
       <Version>6.2.14</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.UI.Xaml">
+      <Version>2.8.6</Version>
+    </PackageReference>
```

Comment:
You also need to add:

```xml
    <PackageReference Include="Microsoft.Web.WebView2">
      <Version>1.0.2903.40</Version>
    </PackageReference>
```

Reply:
Good one. Apparently I was using a local reference.